desktop/src/main/java/bisq/desktop/primary/onboarding/initUserProfile/InitUserProfileController.java

```diff
@@ -75,6 +75,13 @@ void onCreateUserProfile() {
         model.createProfileButtonDisable.set(true);
         model.showProcessingPopup.set(true);
         String profileId = model.profileId.get();
+        // Notes on the first run, when a new user profile is created at startup:
+        // Unfortunately the new profile can only be initialized after the network is fully initialized, which is after
+        // all supported network types have a local address. This translates to longer display times for the processing
+        // popup ("Preparing network node for new identity...").
+        // The good news is, the user is likely to spend some time on the Create Profile popup before clicking Next,
+        // which triggers the new profile initialization. The network layer is being initialized in parallel. This means
+        // in practice, the processing popup won't be shown for long.
```

Comment:
Currently the addresses are created when the user clicks next so the time being on that screen does not help us as the user can change the keypair any time, thus we cannot create the addresses upfront. But we should consider to bake something into the UI which allows us to do that in the background. Maybe we dont need the popup but delay it to the time when the address is actually needed (at create offer later, until then the address shoudld be created, and only if not we need to show the user a "in-progress" screen

Reply:
> Currently the addresses are created when the user clicks next

Correct, but the address creation is relatively fast compared to network init. Also, address creation waits on complete network initialization (e.g. clearnet + tor + i2p).

> ...so the time being on that screen does not help us as the user can change the keypair any time.

In my tests (clearnet + tor) it helped. It didn't eliminate the need to show the popup because as you said address creation takes a bit of time. But it took way less time if some time was spent on that screen vs if user tried to click next right away. 

I spent some time trying to optimize it (make address creation depend on network bootstrap, which advances after first network is ready, vs network init, which completes when all networks are ready). But I couldn't, because the addresses indeed need an initialized network:

https://github.com/bisq-network/bisq2/blob/4f1cf9068e40d41efbb9b0a64a2e6a8fdb952d58/identity/src/main/java/bisq/identity/IdentityService.java#L156-L160

So I added that comment to spare future efforts spent in this direction.

> Maybe we dont need the popup but delay it to the time when the address is actually needed (at create offer later, until then the address shoudld be created, and only if not we need to show the user a "in-progress" screen

Good idea. I'll prepare a PR to do that.

Reply:
Yes, I think best is to remvoe that popup and check if network addresses have been created once they are needed. The identity creating needs its so we need to keep a temporary state until we know all addresses.

Reply:
Should I do that in this PR or a separate one?